besu/src/main/java/org/hyperledger/besu/controller/BesuController.java

```diff
@@ -241,6 +247,9 @@ private BesuControllerBuilder createConsensusScheduleBesuControllerBuilder(
       if (configOptions.isIbft2()) {
         originalControllerBuilder = new IbftBesuControllerBuilder();
       } else if (configOptions.isIbftLegacy()) {
+        // TODO remove this warning once ibf1 has been deprecated
```

Comment:
nit: typo s/ibf1/ibft1...was thinking if anyone tries to search for "ibft1" they wouldn't find this, but probably not worth the extra CI credits to change since it's close to the IbftLegacy.. class anyway!

Reply:
If you put in the constructor as Jason mentioned, then probably don't need the comment since the whole class will presumably be removed